src/effects/VST3/VST3Effect.cpp

```diff
@@ -437,6 +427,8 @@ bool VST3Effect::LoadUserPreset(
    }
    SyncParameters(settings);
 
+   FetchSettings(GetSettings(settings));
```

Comment:
Reviewing the last commit reminds me of `SyncParameters` which happens before this `FetchSettings`.  I think the `SyncParameters` call is another thing that needs to move into `TransferDataToWindow`.

Reply:
Indeed, `SyncParameters` uses the component handler (the handle that will become the Instance object, I think).  So that call doesn't belong in these functions that are supposed to manipulate settings.  It is not one of the "scratch" handles needed to import and export.

So `SyncParameters` does need to move somewhere else.  I'm not sure `TransferDataToWindow` is the right place, but let's begin by making that transformation in this PR at the earlier step.  Maybe we reconsider that later.